mandatory/1-fix-functions.js

```diff
@@ -10,8 +10,7 @@
 
 */
 
-function getMood() {
-  let isHappy = true;
+function getMood(isHappy) {
 
   if (isHappy) {
```

Comment:
Note: In an "early return" pattern you don't need to wrap your second return in an `else` condition.

so you could write your function in following fashion
```
  if (isHappy) {
    return "I am happy";
  } 
  return "I am not happy";

Reply:
Again this is only a suggestion what you have written is correct.